mlir/test/Dialect/LLVMIR/invalid-cg-profile.mlir

```diff
@@ -0,0 +1,14 @@
+// RUN: mlir-translate %s -mlir-to-llvmir | FileCheck %s
+// CHECK: !llvm.module.flags = !{!0, !3}
```

Comment:
Nit: Avoid checks with actual ids, use matchers instead. For example something like this

```suggestion
// CHECK: !llvm.module.flags = !{![[PROF_NODE:[0-9]+]]
```